Backend/testing/test_tts.py

```
@@ -0,0 +1,44 @@
+            "user_text": text
+        })
+        duration = time.time() - start
+        
+        if response.status_code == 200:
+            result = response.json()
+            if result.get('audio_url'):
+                print(f"   ✅ Success in {duration:.1f}s")
+                print(f"   🎵 Audio: {result['audio_url']}")
+                print(f"   📝 Response: {result['response_text'][:60]}...")
+            else:
+                print(f"   ❌ No audio generated")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)